packages/tabs/test/tabs-overflow.test.ts

```diff
@@ -73,7 +93,7 @@ describe('TabsOverflow', () => {
         const el = await fixture<TabsOverflow>(
             html`
                 <sp-tabs-overflow>
-                    <sp-tabs size="m" selected="1" enableTabsScroll=${true}>
+                    <sp-tabs size="m" selected="1" ?enableTabsScroll=${true}>
```

Comment:
This is always setting `true` so we could just do:
```suggestion
                    <sp-tabs size="m" selected="1" enableTabsScroll>
```
However, is it technically correct for this property to be set from an attribute, or should it _always_ be managed by `<sp-tabs-overflow>`?

Reply:
Right, I think it is has no effect when set from an attribute and should be managed by `<sp-tabs-overflow>`. They way I see it, it is just a helper property to let the `<sp-tabs>` know it is wrapper in a `<sp-tabs-overflow>`.

Reply:
The `<sp-tabs-overlay>` sets this value, meaning a consumer of the components shouldn't need to set this value, meaning the story probably shouldn't set a bad example by setting this value...